server/routes/viewModels/taskListModels.js

```diff
@@ -124,8 +124,8 @@ module.exports = (
         addressRejectedInRiskTask: addressUnsuitable,
         addressOrBassOfferedOrUnsuitable: curfewAddressApproved || bassOfferMade || addressUnsuitable,
         licenceUnstarted: stage === 'UNSTARTED',
-        licenceVersionExists: approvedVersion,
-        licenceVaried: !approvedVersion || version > approvedVersion
+        licenceVersionExists: !isEmpty(approvedVersionDetails),
+        licenceVaried: isEmpty(approvedVersionDetails) || version > approvedVersion
```

Comment:
Not following this, a licence is varied if the version is greater than the approved?  I thought the vary version was a separate field?

Reply:
Yes but this version / approved version is the result of sticking them together e.g. 1.2, 1.1
See licenceService line 55 / 56

Reply:
but:
```            const version = `${rawLicence.version}.${rawLicence.vary_version}`;
``` looks like a string to me?  Won't that mean that `1.9 > `1.10`?

Reply:
Correct, need to compare the numbers separately as we are using semantic versioning rather than normal numbers